pkg/crd/markers/validation.go

```diff
@@ -175,6 +183,26 @@ type Default struct {
 	Value interface{}
 }
 
+// +controllertools:marker:generateHelp:category="CRD pruning"
+// PreserveUnknownFields stops the apiserver from pruning fields which are not specified.
+//
+// By default the apiserver drops unknown fields from the request payload
+// during the decoding step.stops the API server. This marker stop the API
```

Comment:
`during the decoding step.stop the API server` looks like a copy paste typo from the next sentence

Reply:
fixed